api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -1766,6 +1766,7 @@ components:
         - next_event_date
         - next_event_type
         - quantity
+        - total_capacity
```

Comment:
[ENT-4431](https://issues.redhat.com/browse/ENT-4431) also mentions product name as an option, let's go ahead and tackle that too while we're updating.

Reply:
Oh I misunderstood that originally. Will update.